lib/parliament/request/base_request.rb

```diff
@@ -101,7 +106,7 @@ class << self
       end
 
       def build_response(net_response)
-        @builder.new(net_response).build
+        @builder.new(response: net_response, decorators: @decorators).build
```

Comment:
Now the decorators are actively ignored using an underscore in the base builder.